README.md

```diff
@@ -26,13 +26,15 @@ For more control over matching, you can provide a match object instead of a simp
 ```yml
 - any: ['list', 'of', 'globs']
   all: ['list', 'of', 'globs']
+  branch: ['list', 'of, 'globs']
```

Comment:
should this and below be `base-branch` / `head-branch`?

Reply:
I haven't pushed my updates to the readme yet

Reply:
it's pushed now 😄